web/src/recent_view_ui.ts

```diff
@@ -1172,7 +1172,10 @@ function recenter_focus_if_off_screen(): void {
         const topic_center_y = (thead_props.bottom + compose_top) / 2;
 
         const topic_element = document.elementFromPoint(topic_center_x, topic_center_y);
-        if (topic_element === null) {
+        if (
+            topic_element === null ||
+            $(topic_element).parents("#recent-view-content-tbody").length === 0
+        ) {
```

Comment:
I don't understand the last commit message; I think the "hence we default" wording is ambigious? Did you mean "hence the existing logic will default to ..."? And then I guess one should explain the fix is to treat the topic element as null if it was that banner, and note we'll select the last row which should be onscreen since it's just above the center element?

Reply:
Sorry, updated the comment to say that.